specs/Separate_conf_files.adoc

```diff
@@ -0,0 +1,141 @@
+= Separate out per-component configuration files
```

Comment:
Change the heading as "Specification for segregating configurations per module wise"